core/domain/exp_domain.py

```diff
@@ -32,13 +32,16 @@
 
 from constants import constants
 from core.domain import change_domain
+from core.domain import exploration_pb2 as Person
 from core.domain import param_domain
 from core.domain import state_domain
 from core.platform import models
 import feconf
 import python_utils
 import utils
 
+from google.protobuf.json_format import Parse as Parse # pylint: disable=import-only-modules
```

Comment:
from google.protobuf import json_format as proto_json_format (say), and then call proto_json_format.Parse in the code.